scripts/bootstrap-build-framework.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+
+if [ ! -d "./make" ]; then
+  version="master"
+  if [ -f ./.dbf ]; then
+    version=$(cat ./.dbf)
+  fi
+
+  docker run -v $(pwd):/app -w /app --env DBFVER=$version deeson/deployer /bin/bash \
+    -c 'git clone -b "$DBFVER" --single-branch --depth 1 https://github.com/teamdeeson/drupal-build-framework.git make \
+    && cd ./make \
+    && rm -Rf .git'
```

Comment:
I know everyone loves docker, but is there really any need to run a git operation inside of a container? Can we really not even assume that developer machines will have git installed?